lib/git_reflow/commands/review.rb

```diff
@@ -5,16 +5,45 @@
   c.arg_name '[destination_branch] - the branch you want to merge your feature branch into'
   c.flag [:t, :title], default_value: 'last commit message'
   c.flag [:m, :message], default_value: 'title'
+  c.switch [:e, :edit]
   c.action do |global_options,options,args|
-    review_options = {
-      'base' => args[0],
-      'title' => global_options[:title],
-      'body' => global_options[:message]
-    }
 
-    review_options['title'] ||=  GitReflow.get_first_commit_message
-    review_options['body']  ||=  review_options['title']
+    commit_msg_file = '/tmp/git_reflow_pr_msg'
 
+    if global_options[:title] || global_options[:message]
+      review_options = {
+        'base' => args[0],
+        'title' => (global_options[:title]   || GitReflow.current_branch),
+        'body' =>  (global_options[:message] || GitReflow.get_first_commit_message)
+      }
+      editor = false
+    else # if edit flag or nothing is provided, open editor
+      editor = true
+    end
+
+    if options[:edit] # if edit we inject title & body into file before opening
+      File.open(commit_msg_file, 'w') do |file|
+        file.write(GitReflow.current_branch)
+        file.write("\n")
+        file.write(GitReflow.get_first_commit_message)
+      end
+    end
+
+    if editor
+      system('nano', commit_msg_file)
+      pr_msg = File.open(commit_msg_file).each_line.map(&:strip).to_a
+      title = pr_msg.shift
+      File.delete(commit_msg_file)
+      review_options = {'base' => args[0],'title' => title,'body' =>  pr_msg.join("\n")}
+    end
+
+    puts "\nReview your PR:\n"
+    puts "--------\n"
+    puts "Title:\n#{review_options['title']}\n\n"
+    puts "Body:\n#{review_options['body']}\n"
+    puts "--------\n"
+    puts "Submit PR? (Y/n):"
+    exit 0 if STDIN.gets.chop.downcase =~ /n/
```

Comment:
we're using the highline gem for input: `GitReflow.review review_options if ask("Submit pull request? (y/n)") =~ /y/i`